src/main/java/Duke.java

```diff
@@ -1,10 +1,88 @@
+import duke.Deadline;
+import duke.Event;
+import duke.Task;
+import duke.Todo;
+import exception.DukeException;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
 public class Duke {
+    static List<Task> tasks = new ArrayList<>();
+    static Boolean notBye = true;
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+//        String logo = " ____        _        \n"
+//                + "|  _ \\ _   _| | _____ \n"
+//                + "| | | | | | | |/ / _ \\\n"
+//                + "| |_| | |_| |   <  __/\n"
+//                + "|____/ \\__,_|_|\\_\\___|\n";
+        String greeting = "Hello I'm Duke\n"
+                + "What can I do for you?";
+        System.out.println(greeting);
+
+        Scanner sc = new Scanner(System.in);
+        String userInput;
+        while(notBye) {
+            userInput = sc.nextLine();
+            try {
+                processUserInput(userInput);
+            }catch (DukeException e){
+                System.out.println(e.toString());
+            }
+            System.out.println("-----------------------------");
+
+        }
+    }
+
+    /**
+     * process the user input and understand the command
+     * @param userInput: value input by a user
+     */
+    private static void processUserInput(String userInput) throws DukeException {
+        String description = null;
+        if(userInput.startsWith("todo")){
```

Comment:
You are handling a few cases based on the user input. Perhaps the nested `if` and several `else if ` could be handled as cases to switch around.